validator_client/src/cli.rs

```diff
@@ -1,478 +1,466 @@
-                    `Small`: 9..=16 \
-                    `Medium`: 17..=64 \
-                    `Large`: 65.. \
-                    These values are used to determine what ordering beacon node fallbacks are used in. \
-                    Generally, `Synced` nodes are preferred over `Small` and so on. \
-                    Nodes in the `Synced` range will tie-break based on their ordering in `--beacon-nodes`. \
-                    This ensures the primary beacon node is prioritised. \
-                    [default: 8,8,48]")
-                .action(ArgAction::Set)
-                .help_heading(FLAG_HEADER)
-                .display_order(0)
-        )
-        .arg(
-            Arg::new("disable-slashing-protection-web3signer")
-                .long("disable-slashing-protection-web3signer")
-                .help("Disable Lighthouse's slashing protection for all web3signer keys. This can \
-                       reduce the I/O burden on the VC but is only safe if slashing protection \
-                       is enabled on the remote signer and is implemented correctly. DO NOT ENABLE \
-                       THIS FLAG UNLESS YOU ARE CERTAIN THAT SLASHING PROTECTION IS ENABLED ON \
-                       THE REMOTE SIGNER. YOU WILL GET SLASHED IF YOU USE THIS FLAG WITHOUT \
-                       ENABLING WEB3SIGNER'S SLASHING PROTECTION.")
-                .action(ArgAction::SetTrue)
-                .help_heading(FLAG_HEADER)
-                .display_order(0)
-        )
-        /*
-         * Experimental/development options.
-         */
-        .arg(
-            Arg::new("web3-signer-keep-alive-timeout")
-                .long("web3-signer-keep-alive-timeout")
-                .value_name("MILLIS")
-                .default_value("20000")
-                .help("Keep-alive timeout for each web3signer connection. Set to 'null' to never \
-                       timeout")
-                .action(ArgAction::Set)
-                .display_order(0)
-        )
-        .arg(
-            Arg::new("web3-signer-max-idle-connections")
-                .long("web3-signer-max-idle-connections")
-                .value_name("COUNT")
-                .help("Maximum number of idle connections to maintain per web3signer host. Default \
-                       is unlimited.")
-                .action(ArgAction::Set)
-                .display_order(0)
-        )
+                and never provide an untrusted URL.",
+        display_order = 0
+    )]
+    pub monitoring_endpoint: Option<String>,
+
+    #[clap(
+        long,
+        value_name = "SECONDS",
+        requires = "monitoring_endpoint",
+        default_value_t = 60,
+        help = "Defines how many seconds to wait between each message sent to \
+                the monitoring-endpoint.",
+        display_order = 0
+    )]
+    pub monitoring_endpoint_period: u64,
+
+    #[clap(
+        long,
+        value_name = "ENABLE_DOPPELGANGER_PROTECTION",
```

Comment:
Should we go with 
`value_name = "BOOLEAN"` for consistency with some of the other flags?

Reply:
makes sense to me